service-base/src/test/java/org/eclipse/hono/service/VertxBasedHealthCheckServerTest.java

```diff
@@ -128,7 +129,7 @@ void testHealthCheckServerWithSecureEndpoint(final Vertx vertx, final VertxTestC
             .map(ok -> getWebClient(vertx, server, true))
             .compose(httpClient -> checkHealth(ctx, httpClient, "/liveness"))
             .compose(httpClient -> checkHealth(ctx, httpClient, "/readiness"))
-            .onComplete(ctx.completing());
+            .onComplete(ctx.succeeding(v -> testsDone.flag()));
```

Comment:
why should this change be necessary?

Reply:
Because of the `registerHealthChecks` invocation before - see obvious.

Reply:
I see